ocs_ci/ocs/ui/mcg_ui.py

```diff
@@ -0,0 +1,310 @@
+import logging
+
+from selenium.webdriver.support.wait import WebDriverWait
+
+from ocs_ci.ocs.ui.base_ui import PageNavigator
+from ocs_ci.ocs.ui.views import locators
+from ocs_ci.utility.utils import get_ocp_version
+
+logger = logging.getLogger(__name__)
+
+
+class MCGStoreUI(PageNavigator):
+    """
+    A class representation for abstraction of BS-related OpenShift UI actions
+
+    """
+
+    def __init__(self, driver):
+        super().__init__(driver)
+        self.wait = WebDriverWait(self.driver, 30)
+        ocp_version = get_ocp_version()
+        self.ocs_loc = locators[ocp_version]["ocs_operator"]
+        self.mcg_stores = locators[ocp_version]["mcg_stores"]
+
+    def create_store_ui(self, kind, store_name, secret_name, target_bucket):
```

Comment:
should we maybe set `kind` with a default value?

Reply:
I don't feel it'd be right; The user _should_ be clear and intentional when creating MCG stores. There's no really "default" store kind